src/styles/root.scss

```diff
@@ -0,0 +1,15 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$gray-elements: #f3f3f3;
+$background-main: #fff;
+$starsCount: 1,2,3,4,5;
+
+body,
+p,
+h2,
+h3 {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
You don't need to set the font-family to the p, h2, h3, etc.
Set it only to the tag body